server/src/main/java/com/skillmagnet/User/User.java

```diff
@@ -1,6 +1,10 @@
 package com.skillmagnet.User;
 
+import java.time.LocalDateTime;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
 import jakarta.persistence.*;
+import org.springframework.security.crypto.argon2.Argon2PasswordEncoder;
 
 @Entity
```

Comment:
:memo: Note this [PR](https://github.com/nickerick/skill-magnet/pull/25). Can add `@Data` above class and will auto add getters/setters.

Reply:
Thats pretty nifty